ga4gh/datamodel/__init__.py

```diff
@@ -112,9 +113,13 @@ def _setAccessTimes(self, directoryPath):
         directoryPath.
         """
         # ctime is in seconds, and we want milliseconds since the epoch
-        ctimeInMillis = int(os.path.getctime(directoryPath) * 1000)
-        self._creationTime = ctimeInMillis
-        self._updatedTime = ctimeInMillis
+        if os.path.exists(directoryPath):
```

Comment:
I don't think this works as intended --- the directory will always exist, even if the constituent files are remote. It's arguable what  the ctime of the directory actually means here, but this is a bigger issue. I would just leave this as it is.